plugins/include/open62541/plugin/pubsub_realtime_etf.h

```diff
@@ -0,0 +1,74 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information.
+ *
+ *    Copyright (c) 2019-2020 Kalycito Infotech Private Limited
+ */
+
+#ifndef PUBSUB_REALTIME_ETF_H_
+#define PUBSUB_REALTIME_ETF_H_
+
+#ifdef UA_ENABLE_PUBSUB_REALTIME_PUBLISH_ETF
+
+#include "ua_pubsub.h"
+#include "time.h"
+#include <open62541/plugin/log_stdout.h>
+#include <linux/types.h>
+#ifdef UA_ENABLE_PUBSUB_ETH_UADP
+#include <open62541/plugin/pubsub_ethernet.h>
+#endif
+
+#define   CLOCKIDENTITY                                   CLOCK_TAI
+#define   MULTICAST_ADDRESS                               "224.0.0.32"

```

Comment:
The multicast address is part of the example, not the general functionality.
It should notbe in /plugins/include. Move it to /examples/pubsub_realtime.

Do not include <linux/types.h> and <open62541/plugin/log_stdout.h> here.
This should be part of the implementation. But not part of the interface.